src/types.ts

```diff
@@ -75,6 +75,7 @@ export const chainSchema = z
     explorer: chainExplorerSchema,
     blockTimeMs: z.number().positive(),
     hardhatConfigOverrides: hardhatConfigOverrides.optional(),
+    skipProviderCheck: z.boolean().optional(), // For chains not supporting dAPIs
```

Comment:
hah this was simpler than what I was picturing 😅

Reply:
Haha me too